lib/src/container/mod.rs

```diff
@@ -333,10 +333,15 @@ impl<'a> ManifestDiff<'a> {
 /// as root, then a default isolation of running the process via `nobody` will be applied.
 pub fn merge_default_container_proxy_opts(
     config: &mut containers_image_proxy::ImageProxyConfig,
+    transport: Transport,
```

Comment:
This API is public though, so this change would be a semver break; not sure we want to do another semver break cycle right now.

We could add a new public API though.

(Also then we'll need to change rpm-ostree to call it)

Reply:
Took the previous approach